manager/src/main.rs

```diff
@@ -168,51 +215,132 @@ fn sidecar(config: &Mapping, addr: &str) -> Result<(), Box<dyn Error>> {
+let mempool_info = std::process::Command::new("bitcoin-cli")
+    .arg("-conf=/root/.bitcoin/bitcoin.conf")
+    .arg("getmempoolinfo")
+    .output()?;
+
+if mempool_info.status.success() {
+    let mempool_data: serde_json::Value = serde_json::from_slice(&mempool_info.stdout)?;
+
+    let max_mempool = mempool_data["maxmempool"].as_u64().unwrap_or(0) as f64 / 1024_f64.powf(2.0); // Convert to MB
+    let mempool_usage = mempool_data["usage"].as_u64().unwrap_or(0) as f64 / 1024_f64.powf(2.0); // Convert to MB
+    let mempool_percent = if max_mempool > 0.0 {
+        (mempool_usage / max_mempool) * 100.0
+    } else {
+        0.0
+    };
+    let tx_count = mempool_data["size"].as_u64().unwrap_or(0); // Number of transactions
+
+    // Insert the simplified mempool summary
+    stats.insert(
+        Cow::from("Mempool Summary"),
+        Stat {
+            value_type: "string",
+            value: format!(
+                "{:.2}/{:.2} MB ({:.2}%), {} Transactions",
+                mempool_usage, max_mempool, mempool_percent, tx_count
+            ),
+            description: Some(Cow::from("Summary of current mempool usage and transaction count")),
+            copyable: false,
+            qr: false,
+            masked: false,
+        },
+    );
+} else {
+    eprintln!(
+        "Error retrieving mempool info: {}",
+        std::str::from_utf8(&mempool_info.stderr).unwrap_or("UNKNOWN ERROR")
+    );
+}
+
+    // Existing code for blockchain and network info retrieval continues here...
+    let info_res = std::process::Command::new("bitcoin-cli")
+        .arg("-conf=/root/.bitcoin/bitcoin.conf")
+        .arg("getblockchaininfo")
+        .output()?;
+    if info_res.status.success() {
+        let info: ChainInfo = serde_json::from_slice(&info_res.stdout)?;
+        // Consolidated block height and sync progress information
+        // Extract values for current synced blocks, total headers, and sync progress percentage
+        let current_blocks = info.blocks;
+        let total_headers = info.headers;
+        let sync_progress = if current_blocks < total_headers {
+            100.0 * info.verificationprogress
+        } else {
+            100.0
+        };
+
+        // Insert the simplified blockchain sync summary
         stats.insert(
-            Cow::from("Sync Progress"),
+            Cow::from("Blockchain Sync Summary"),
             Stat {
                 value_type: "string",
-                value: if info.blocks < info.headers {
-                    format!("{:.2}%", 100.0 * info.verificationprogress)
-                } else {
-                    "100%".to_owned()
-                },
-                description: Some(Cow::from(
-                    "The percentage of the blockchain that has been verified",
-                )),
+                value: format!(
+                    "{}/{} ({:.2}%)",
+                    current_blocks, total_headers, sync_progress
+                ),
```

Comment:
This is a great improvement to make the Properties more concise. Nicely done. However I would add some spacing here for better readability.

```suggestion
                value: format!(
                    "{} / {} ({:.2}%)",
                    current_blocks, total_headers, sync_progress
                ),
```